metrics-operator/main.go

```diff
@@ -30,9 +30,7 @@ import (
 	"github.com/keptn/lifecycle-toolkit/klt-cert-manager/pkg/certificates"
 	certCommon "github.com/keptn/lifecycle-toolkit/klt-cert-manager/pkg/common"
 	certwebhook "github.com/keptn/lifecycle-toolkit/klt-cert-manager/pkg/webhook"
-	metricsv1alpha1 "github.com/keptn/lifecycle-toolkit/metrics-operator/api/v1alpha1"
-	metricsv1alpha2 "github.com/keptn/lifecycle-toolkit/metrics-operator/api/v1alpha2"
-	metricsv1alpha3 "github.com/keptn/lifecycle-toolkit/metrics-operator/api/v1alpha3"
+	metricsv1beta1 "github.com/keptn/lifecycle-toolkit/metrics-operator/api/v1beta1"
```

Comment:
this could be refactored to just `metricsapi` like in most other files, now that only one version is used in the file